sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/DynamicDestinations.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.io.iceberg;
+package org.apache.beam.sdk.io.iceberg;
```

Comment:
As discussed offline, will implement custom serialization for `OneTableDynamicDestinations` and make translation fail for other DynamicDestination classes

Reply:
Can we simply omit the dynamic destinations parameter for now? I'd really like to avoid schemas with java-only types.

Reply:
I don't think Java only types are completely avoidable (at least without a significant refactoring). I think the main concern is failing to deserialize objects when upgrading the Java transform  in the future. So we implemented `Externalizable` and added customer serializations that should be future proof for Java only types used here.

Reply:
After discussing offline, we decided it's best to not worry about serializing Java objects.

Instead of serializing at the IcebergIO level, we decided to serialize at the Iceberg SchemaTransform level (which doesn't include DynamicDestinations)